### PR TITLE
Refactor/move core to upm layout

### PR DESCRIPTION
**Pull request by andrew-hirata-playeveryware**
Move all the core files to the new UPM layout directory.

### CODE REVIEW

File: etc/PackageConfigurations/eos_package_description.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
                              
         {"src": "Assets/Plugins/*",                                "dest": "Runtime/"                             },
         {"src": "Assets/Plugins/Source/Core.meta",                 "dest": "Runtime/"                             },
-        {"src": "Assets/Plugins/Source/Core/*",                    "dest": "Runtime/Core/"                        },
-        {"src": "Assets/Plugins/Source/Core/Utility/*",            "dest": "Runtime/Core/Utility/"                },
-        {"src": "Assets/Plugins/Source/Core/Utility/Extensions/*", "dest": "Runtime/Core/Utility/Extensions/"     },
+        {"src": "com.playeveryware.com/Runtime/Core/*",                    "dest": "Runtime/Core/"                        },
```
Comment (WispyMouse) [inline etc/PackageConfigurations/eos_package_description.json:17]: Above this, it is clear that we're maintaining "Assets/Plugins/Source/Core.meta". Should there be an associated .meta file for "com.playeveryware.com/Runtime/Core/", or is this outside of the folder structure requiring meta files?

**Review by WispyMouse — APPROVED** [review]
LGTM. No cataclysmic failures from opening and building the project.

File: com.playeveryware.eos/Runtime/Core/com.playeveryware.eos.core.asmdef
Comment (paulhazen) [inline com.playeveryware.eos/Runtime/Core/com.playeveryware.eos.core.asmdef:1]: Does it still make sense (in the context of these changes) for the assembly to be named thusly? It's not clear to me what the intersections and interplay are between assembly naming conventions, directory structure, and UPM layout.

**Review by paulhazen — APPROVED** [review]

File: etc/PackageConfigurations/eos_package_description.json (hunk 1 above)
Comment (paulhazen) [reply etc/PackageConfigurations/eos_package_description.json:17]: Nice catch! This does need to be fixed.

File: com.playeveryware.eos/Runtime/Core/com.playeveryware.eos.core.asmdef
Comment (andrew-hirata-playeveryware) [reply com.playeveryware.eos/Runtime/Core/com.playeveryware.eos.core.asmdef:1]: In the context of this change yes: that would be a different kind of refactor to change it.

**Pull request merged**